boards/arm/nucleo_h743zi/nucleo_h743zi.dts

```diff
@@ -64,6 +64,11 @@
 	status = "okay";
 };
 
+&usbotg_fs {
```

Comment:
Please update boards documentation

Reply:
I've updated the doc and board yaml files to indicate `usb_device` support.